components/style/properties/longhand/column.mako.rs

```diff
@@ -31,8 +31,7 @@ ${helpers.predefined_type(
 
 
 <%
-# FIXME(#20498): Servo should support percentages in column-gap.
-col_gap_type = "NonNegativeLengthOrPercentageOrNormal" if product == "gecko" else "NonNegativeLengthOrNormal"
+col_gap_type = "NonNegativeLengthOrPercentageOrNormal"
```

Comment:
You can get rid of this variable, and pass `"length::NonNegativeLengthOrPercentageOrNormal"` directly as the second argument to `predefined_type`, below.

Reply:
You mean something like this?

```rust
${helpers.predefined_type(
    "column-gap",
    "length::NonNegativeLengthOrPercentageOrNormal",
    "Either::Second(Normal)",
    extra_prefixes="moz",
    servo_pref="layout.columns.enabled",
    animation_value_type="NonNegativeLengthOrPercentageOrNormal",
    spec="https://drafts.csswg.org/css-multicol/#propdef-column-gap",
    servo_restyle_damage = "reflow",
)}
```

Reply:
Yes.

Reply:
Done :+1: